Formula/poac.rb

```diff
@@ -0,0 +1,37 @@
+class Poac < Formula
+  desc "Package manager for C++"
+  homepage "https://github.com/poacpm/poac"
+  url "https://github.com/poacpm/poac.git",
+    tag:      "0.3.4",
+    revision: "f905a175ccd1b9ad94041e1e0a29d12d92e8308d"
+  license "Apache-2.0"
+  head "https://github.com/poacpm/poac.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+  depends_on macos: :big_sur # C++20
+  depends_on "openssl@3"
+
+  on_linux do
+    depends_on "gcc"
+  end
+  fails_with gcc: "5" # C++20
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make", "install"
+    end
```

Comment:
```suggestion
    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
    system "cmake", "--build", "build"
    system "cmake", "--install", "build"
```

Reply:
@carlocab 

I used a shorter command, but do you prefer what you suggested?

```rb
system "cmake", "--build", "build", "--target", "install"
```